private/model/api/operation.go

```diff
@@ -239,6 +247,12 @@ func (c *{{ .API.StructName }}) {{ .ExportedName }}PagesWithContext(` +
 // GoCode returns a string of rendered GoCode for this Operation
 func (o *Operation) GoCode() string {
 	var buf bytes.Buffer
+
+	if len(o.OutputRef.Shape.EventStreamsMemberName) != 0 {
+		// TODO need better was of updating protocol unmarshalers
+		o.API.imports["github.com/aws/aws-sdk-go/private/protocol/rest"] = true
```

Comment:
the old protocol will still be referenced in the file because of the swap out of the old unmarshaler for the rest one. So both packages will always be referenced.

```go
req.Handlers.Unmarshal.Swap(restxml.UnmarshalHandler.Name, rest.UnmarshalHandler)
```